### PR TITLE
Review of FAQ sections with suggested improvements

### PR DESCRIPTION
**Pull request by danimtb**
This is the new PR with the changes made by @adiatzmony in #1042 but with the latest changes from master branch.

I have reverted some commits and added a small review about workspaces (removing ``[EXPERIMENTAL]`` from title and so on)

### CODE REVIEW

File: faq/using.rst [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+:ref:`packaging getting started guide<packaging_getting_started>`. The main difference is that a package recipe is typically much simpler.
+There are different approaches depending on if you want Conan to run the library unit tests while creating the package or not. Full details are described
+:ref:`in this how-to guide<header_only>`.
 
 When to use settings or options?
 --------------------------------
 
-While creating a package you might want to add different configurations and variants of the package. There are 2 main inputs that define
-packages: settings and options. Read about them in :ref:`this section<settings_vs_options>`
+While creating a package, you may want to add different configurations and variants of the package. There are 2 main inputs that define
```
Comment (jgsogo) [inline faq/using.rst]:
```suggestion
While creating a package, you may want to add different configurations and variants of the package. There are two main inputs that define
```

File: developing_packages/workspaces.rst [hunk 2]
```diff
@@ -70,31 +69,30 @@ Next, run a :command:`conan install` as usual, using a *build* folder to output
...
 
 .. code-block:: bash
 
     $ conan search
     There are no packages
 
-Also, all the generated *conanbuildinfo.cmake* files for the dependencies are installed in the *build* folder. You can inspect them to check
-that the paths they define for their dependencies are user folders. They don't point to the local cache.
+Also, all the generated *conanbuildinfo.cmake* for each dependencies are installed in the *build* folder. You can inspect them to check
+that the paths they define for their dependencies, are user folders, not pointing to the local cache.
```
Comment (jgsogo) [inline developing_packages/workspaces.rst]:
```suggestion
that the paths they define for their dependencies are user folders, not pointing to the local cache.
```

File: developing_packages/workspaces.rst [hunk 3]
```diff
@@ -70,31 +69,30 @@ Next, run a :command:`conan install` as usual, using a *build* folder to output
...
+Note that nothing will really be installed in the local cache, all the dependencies are resolved locally:
 
 .. code-block:: bash
 
     $ conan search
     There are no packages
 
-Also, all the generated *conanbuildinfo.cmake* files for the dependencies are installed in the *build* folder. You can inspect them to check
-that the paths they define for their dependencies are user folders. They don't point to the local cache.
+Also, all the generated *conanbuildinfo.cmake* for each dependencies are installed in the *build* folder. You can inspect them to check
```
Comment (jgsogo) [inline developing_packages/workspaces.rst]: Each dependency generates one *conanbuildinfo.cmake* file, all those files are installed in the *build* folder. I think this is not what the sentence says.

Comment (danimtb) [inline developing_packages/workspaces.rst]:
```suggestion
Also, each dependency generates one *conanbuildinfo.cmake* file and all of them are installed in the *build* folder. You can inspect them to check
```

**Comment by jgsogo** [comment]
@danimtb, this branch is to be removed, isn't it?

**Comment by danimtb** [comment]
Yes. Done! Thanks for the reminder

**Pull request merged**